pycontw/pycon.tw
# Add a 45 min option to the duration in the talk proposal form

## Add a tick box to the proposal form indicating 1st-time speaker

## Description
Support program team's new requirement. 
Add a 45 min option to the duration in the talk proposal form that give the speaker more option to choose.

## Steps to Test This Pull Request
Steps to reproduce the behavior:
1. Go to 'MY PYCON'
2. Click on 'proposals' tab
4. Click on '+New Talk Proposal'
3. Scroll down
4. See tick box of *duration*

## More Information
**Screenshots**
- Frontend

![image](https://user-images.githubusercontent.com/20749936/106757857-d0782280-666b-11eb-87c4-202f2c4350ba.png)

- Django Administration
![image](https://user-images.githubusercontent.com/20749936/106757976-f271a500-666b-11eb-9e13-e92da576645c.png)
